### ISSUE TITLE
Organize imports breaks multicatch

### ISSUE THREAD

**Issue by tivervac** [eclipse/xtext-xtend#508]
With Xtext 2.14 `Organize imports` breaks multicatch expressions.
```
import java.io.FileNotFoundException

class Test {
	def void test() {
		try {
			method();
		} catch (FileNotFoundException | IllegalArgumentException e) {
			
		}
	}

	def void method() {
		throw new UnsupportedOperationException("TODO: auto-generated method stub")
	}
}
```
Turns into
```
import java.io.FileNotFoundException

class Test {
	def void test() {
		try {
			method();
		} catch (Object e) {
			
		}
	}

	def void method() {
		throw new UnsupportedOperationException("TODO: auto-generated method stub")
	}
}
```

**Comment by cdietrich** [comment]
@szarnekow can it be we need to adapt the type usage collector?

**Comment by szarnekow** [comment]
Yes, apparently we do have to do that.

**Comment by cdietrich** [comment]
see e.g. org.eclipse.xtext.xbase.imports.TypeUsageCollector.collectAllReferences(EObject)

**Comment by cdietrich** [comment]
Fixed in 2.15
